index.html

```diff
@@ -257,6 +258,13 @@ <h4>The Top-Level <code>&lt;math&gt;</code> Element</h4>
             attribute is absent or has an invalid value, the User Agent
             stylesheet treats it the same as <code>inline</code>.
           </p>
+
+            
+	  <p>Systems implementing MathML Core may ignore
```

Comment:
nit: MathML Core tries to use "the user agent" as done in HTML5.

Reply:
yes but we should normalise on the

> This specification does not define any observable behavior that is specific to the fence and separator attributes. 

wordig as you note elsewhere.  So this can probably go.